tech-far-hub/src/components/tfh.scss

```diff
@@ -167,4 +167,16 @@ li.usa-card {
       margin-top: 1rem;
     }
   }
+  .tfh-resources-landing .tfh-lifecycleResouces .usa-card__body {
+    min-height: 300px;
+    ul {
+      li {
+        flex: 250px;
+        margin-top: 2rem;
+        a {
```

Comment:
leveraging nested selectors is awesome as it reduces classnames. I've noticed that for J40 (rapidly changing copy), sometimes this can lead to less flexibility. Might not be the case on TFH